### DISCUSSION TITLE
Just to show why HoMM2 doesn't need any rebalance

### DISCUSSION THREAD

**Discussion by ihhub** [ihhub/fheroes2#5888] [General]
Who said that Warlocks are better than Knights?! Quality of  video is set by Windows so I couldn't do much.

https://user-images.githubusercontent.com/19829520/192280606-b2fcbff9-1c76-410e-b2cc-ee802b4cfd6b.mp4

**Comment by LeHerosInconnu** [comment]
Hello @ihhub,

Yes, one has to look at the balance of the original game as a whole, also taking into account the size of the map.
For me, all castles have a chance, and on all map sizes.
You just have to play according to the strengths and weaknesses of each castle.

**Reply by zenseii** [reply]
Hi, @LeHerosInconnu.

I believe @ihhub was being ironic since this claim about game balance and warlock being the best faction is so widespread.

Still, even if knight won in the video the bigger point, besides showing off the new ai control feature, is to show that the AI plays quite well, but still has some ways to go.
